packages/types/src/Null.ts

```diff
@@ -10,10 +10,16 @@ import { Codec } from './types';
  * Implements a type that does not contain anything (apart from `null`)
  */
 export default class Null implements Codec {
+  /**
+   * @description The length of the value when encoded as a Uint8Array
+   */
   get encodedLength (): number {
     return 0;
   }
 
+  /**
+   * @description Returns a hex string representation of the value
```

Comment:
does @inheritdoc work here? In which case you don't need to put the same desc everywhere.